venues/OpenReview.net/Support/init.py

```diff
@@ -70,13 +69,13 @@
         'description': 'By when do authors need to submit their manuscripts? Please submit in the following format: YYYY/MM/DD HH:MM(e.g. 2019/01/31 23:59)',
         'order': 8
     },
-    'Conference Start Date': {
-        'description': 'What is the start date of conference itself? Please submit in the following format: YYYY/MM/DD (e.g. 2019/01/31)',
+    'Venue Start Date': {
+        'description': 'What is the start date of venue itself? Please submit in the following format: YYYY/MM/DD (e.g. 2019/01/31)',
```

Comment:
Changed the text here to what Zach suggested. I agree it is slightly weird but I feel that it will do for now.
As for saving the venue type, what use do you envision for that @zbialecki ?